notebooks/en/llm_judge.ipynb

```diff
@@ -0,0 +1,744 @@
+{
```

Comment:
<p>Let's remove this output, it doesn't add much value.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
✅